examples/platforms/jn5189/Makefile.am

```diff
@@ -0,0 +1,129 @@
+    system.c                                                                                         \
+    uart.c                                                                                           \
+    settings_jn5189.c                                                                                \
+    @top_builddir@/third_party/nxp/JN5189/drivers/fsl_assert.c                                       \
+	@top_builddir@/third_party/nxp/JN5189/crypto/aes_alt.c                                           \
+    @top_builddir@/third_party/nxp/JN5189/crypto/ksdk_mbedtls.c                                      \
+    $(NULL)
+
+libopenthread_jn5189_sdk_a_SOURCES                                                                 = \
+    @top_builddir@/third_party/nxp/JN5189/startup_JN5189.c                                           \
+    @top_builddir@/third_party/nxp/JN5189/system_JN5189.c                                            \
+    @top_builddir@/third_party/nxp/JN5189/dk6_jn5180/pin_mux.c                                       \
+    @top_builddir@/third_party/nxp/JN5189/dk6_jn5180/clock_config.c                                  \
+    @top_builddir@/third_party/nxp/JN5189/drivers/fsl_gpio.c                                        \
+    @top_builddir@/third_party/nxp/JN5189/drivers/fsl_clock.c                                        \
+    @top_builddir@/third_party/nxp/JN5189/drivers/fsl_ctimer.c                                       \
+    @top_builddir@/third_party/nxp/JN5189/drivers/fsl_wtimer.c                                       \
+    @top_builddir@/third_party/nxp/JN5189/drivers/fsl_flash.c                                        \
+    @top_builddir@/third_party/nxp/JN5189/drivers/fsl_usart.c                                        \
+    @top_builddir@/third_party/nxp/JN5189/drivers/fsl_rng.c                                          \
+    @top_builddir@/third_party/nxp/JN5189/drivers/fsl_flexcomm.c                                     \
+    @top_builddir@/third_party/nxp/JN5189/drivers/fsl_reset.c                                        \
+    @top_builddir@/third_party/nxp/JN5189/drivers/fsl_power.c                                        \
+    @top_builddir@/third_party/nxp/JN5189/drivers/fsl_debug_console.c                                \
+    @top_builddir@/third_party/nxp/JN5189/drivers/aes/fsl_aes.c                                      \
+    @top_builddir@/third_party/nxp/JN5189/drivers/sha/fsl_sha.c                                      \
+    $(NULL)
+
+libopenthread_jn5189_plat_a_SOURCES                                                                = \
+    $(PLATFORM_SOURCES)                                                                              \
+    $(NULL)
+
+PRETTY_FILES                                                                                       = \
+    $(PLATFORM_SOURCES)                                                                              \
```

Comment:
There are some `third_party/nxp` files included in `PLATFORM_SOURCES`. Do you really want OpenThread's make pretty running on `third_party/nxp` files?

Reply:
No, I reverted the files from third_party/nxp.

Reply:
The problem now is that the Travis CI build is failing due to pretty check errors caused by third_party/nxp files. How could I fix that?

Reply:
#4660 should help address this.

Reply:
@doru91, I think you'll need to run `./script/make-pretty clang` again to resolve the code format issues.

Reply:
Thanks, @jwhui. I run again the make-pretty script and now the Travis build is fixed.

Reply:
@jwhui, is there any other change needed in order to have this pull request approved?

Reply:
Had to resolve a couple conflicts in `configure.ac`. This should be ready once the build checks pass.